source/MRCuda/MRCudaFastWindingNumber.cuh

```diff
@@ -40,5 +40,10 @@ void fastWindingNumberFromGridKernel( int3 gridSize, float3 minCoord, float3 vox
                                       const Dipole* dipoles, const Node3* nodes, const float3* meshPoints, const FaceToThreeVerts* faces,
                                       float* resVec, float beta );
 
+// calls fast winding number for each point in three-dimensional grid
+void fastWindingNumberFromGridWithDistancesKernel( int3 gridSize, float3 minCoord, float3 voxelSize, Matrix4 gridToMeshXf,
```

Comment:
better name somehow `signedDistanceKernel`, and in the comments specify that it uses winding number inside